src/popup.tsx

```diff
@@ -49,187 +46,158 @@ import ContactSettings from "~routes/popup/settings/contacts/[address]";
-                    <SubscriptionManagement id={params?.id} />
-                  )}
-                </Route>
-                <Route path="/subscriptions/:id/payment">
-                  {(params: { id: string }) => (
-                    <SubscriptionPayment id={params?.id} />
-                  )}
-                </Route>
-                <Route path="/transactions" component={Transactions} />
-                <Route path="/notifications" component={Notifications} />
-                <Route path="/notification/:id">
-                  {(params: { id: string }) => (
-                    <MessageNotification id={params?.id} />
-                  )}
-                </Route>
-                <Route path="/tokens" component={Tokens} />
-                <Route path="/token/:id">
-                  {(params: { id: string }) => <Asset id={params?.id} />}
-                </Route>
-                <Route path="/collectibles" component={Collectibles} />
-                <Route path="/collectible/:id">
-                  {(params: { id: string }) => <Collectible id={params?.id} />}
-                </Route>
-                <Route path="/transaction/:id/:gateway?">
-                  {(params: { id: string; gateway?: string }) => (
-                    <Transaction id={params?.id} gw={params?.gateway} />
-                  )}
-                </Route>
-                <Route path="/send/confirm/:token/:qty/:recipient/:message?">
-                  {(params: { token: string; qty: string }) => (
-                    <Confirm
-                      tokenID={params?.token}
-                      qty={Number(params?.qty || "0")}
-                    />
-                  )}
-                </Route>
-                <Route path="/send/recipient/:token/:qty/:message?">
-                  {(params: {
-                    token: string;
-                    qty: string;
-                    message?: string;
-                  }) => (
-                    <Recipient
-                      tokenID={params?.token}
-                      qty={params?.qty || "0"}
-                      message={params?.message || ""}
-                    />
-                  )}
-                </Route>
-                <NavigationBar />
-              </HistoryProvider>
-            </Router>
-          </Wrapper>
-        </ExpandedViewWrapper>
-      </HardwareWalletTheme>
-    </Provider>
-  );
-}
-
-const HideScrollbar = createGlobalStyle<{ expanded?: boolean }>`
-  * {
-    scrollbar-width: none;
+  let content: React.ReactElement = null;
 
-    &::-webkit-scrollbar {
-      display: none
-    }
+  if (initialScreenType === "cover") {
+    content = <Page />;
+  } else if (initialScreenType === "locked") {
+    content = (
+      <Page>
+        <Unlock />
+      </Page>
+    );
+  } else if (initialScreenType === "generating") {
+    // This can only happen in the embedded wallet:
+    content = (
+      <Page>
+        <p>Generating Wallet...</p>
```

Comment:
Placeholder for the embedded wallet. This does nothing right now, but I'll update it in the other PR once this one is merged.